lua/autorun/sh_votestuff.lua

```diff
@@ -0,0 +1,25 @@
+// Shared utils for votestuff by Ross;
+
+function RFormatTime(time)
```

Comment:
https://wiki.facepunch.com/gmod/string.FormattedTime

Reply:
> https://wiki.facepunch.com/gmod/string.FormattedTime

I've done this function before this appeared in gmod I think. I'll use it, thanks for point